pkg/types/baremetal/validation/platform_test.go

```diff
@@ -47,6 +47,8 @@ func TestValidatePlatform(t *testing.T) {
 				BootstrapProvisioningIP: "172.22.0.2",
 				ExternalBridge:          "br0",
 				ProvisioningBridge:      "br1",
+				BootstrapOSImage:        "http://192.168.111.1/images/qemu.x86_64.qcow2.gz?sha256=3b5a882c2af3e19d515b961855d144f293cab30190c2bdedd661af31a1fc4e2f",
+				ClusterOSImage:          "http://192.168.111.1/images/metal.x86_64.qcow2.gz?sha256=340dfa4d92450f2eee852ed1e2d02e3138cc68d824827ef9cf0a40a7ea2f93da",
```

Comment:
these fields are optional, so i would expect validation to be valid even when these are empty.

so 2 different test-cases no provided, provided valid.

Reply:
Done. Thanks for the suggestion.